app/controllers/concerns/helpers.rb

```diff
@@ -75,12 +65,12 @@ def build_board_view(game, row_size, show_bombs=nil)
     game.board_positions.each_slice(row_size).map.with_index do |row, row_index|
       row.map.with_index do |cell, cell_index|
         cell_position = row_index * row_size + cell_index
-        content = board_array[cell_position]
-        submit_btn = board_array[cell_position]
-        cell_class = game.board_positions[cell_position].status == 'revealed' ? 'cell-submit active' : 'cell-submit'
-        form_status =
-        game.board_positions[cell_position].status == 'revealed' ? true : false
-        [cell_position, content, submit_btn, cell_class, form_status]
+        submit_btn = board_array[cell_position] == 'BF' ? bomb : board_array[cell_position]
+        cell_class ='cell-submit'
```

Comment:
One benefit of a model-view-controller design is that all the view-related logic (like CSS class names) can stay in the view class, hidden from the rest of its collaborators. On this line, it looks like CSS class names are leaking into the controller. Is there a way to move the specific class names back into the view, but provide the view the same information (whether a cell is active or not)? 

In some complex cases where you need to do more formatting, you might want to use a "presenter" object. Look up the "presenter pattern" for more information on this strategy.